src/globus_sdk/services/transfer/client.py

```diff
@@ -274,6 +274,52 @@ def update_endpoint(
         log.info(f"TransferClient.update_endpoint({endpoint_id}, ...)")
         return self.put(f"endpoint/{endpoint_id}", data=data, query_params=query_params)
 
+    def set_subscription_id(
+        self, collection_id: UUIDLike, subscription_id: UUIDLike
```

Comment:
`"DEFAULT"` is an allowable value. I believe since `UUIDLike` just unions `UUID` and `str` types, it doesn't pose a problem, though I do wonder if it'd be helpful to add an explicit literal here (and/or a note to the `:param` below) indicating that it's a valid, "special" value?

Also, though the docs don't mention it, is `null` valid for `subscription_id` (for instance, to clear a subscription)?

Reply:
Good question about `null`. I should test and we can decide based on what it does.

I considered taking extra steps with `subscription_id="DEFAULT"`. I think I'm -0 on changing the annotation -- it doesn't really harm anything, but I don't know that it's helpful -- but doc is where I'm a bit less confident of my judgement to omit it. Since you've asked, I'll add it.

Reply:
```
$ globus api transfer PUT /endpoint/$GCP_ID/subscription --body '{"subscription_id": null}'
```

worked.

So yes, we should allow `None`. :slightly_smiling_face:

Reply:
I tried a variant of this which used `Literal["DEFAULT"]` even though that's a subtype of `UUIDLike`. LMK if you think it's nicer or not -- I think of this as a little experiment in "how does this look?"

Reply:
Yeah, I think you're a -0 and I'm a +0. I think it's a little nicer (I am specifically thinking about this as used by a type-savvy IDE) but I will defer to you if you still dislike it. I'll approve now for expediency, and will be fine with either outcome.

Reply:
The law of "that's the way it is in in the PR" wins. Merging :wink: